source/material_model/rheology/strain_dependent.cc

```diff
@@ -360,6 +396,32 @@ namespace aspect
 
       }
 
+      template <int dim>
+      double
+      StrainDependent<dim>::
+      calculate_strain_healing(const MaterialModel::MaterialModelInputs<dim> &in,
+                               const double e_ii,
+                               const unsigned int j) const
+      {
+        const double reference_temperature = this->get_adiabatic_surface_temperature();
```

Comment:
We need a check somewhere to make sure that `get_adiabatic_surface_temperature()` is not zero, as you will have divide by zero scenario below.